amqp/src/main/scala/akka/stream/alpakka/amqp/scaladsl/AmqpSink.scala

```diff
@@ -3,23 +3,33 @@
  */
 package akka.stream.alpakka.amqp.scaladsl
 
-import akka.NotUsed
-import akka.stream.alpakka.amqp.{AmqpSinkSettings, AmqpSinkStage, OutgoingMessage}
+import akka.{Done, NotUsed}
+import akka.stream.alpakka.amqp._
 import akka.stream.scaladsl.Sink
 import akka.util.ByteString
 
+import scala.concurrent.Future
+
 object AmqpSink {
 
   /**
    * Scala API: Creates an [[AmqpSink]] that accepts ByteString elements.
    */
-  def simple(settings: AmqpSinkSettings): Sink[ByteString, NotUsed] =
+  def simple(settings: AmqpSinkSettings): Sink[ByteString, Future[Done]] =
     apply(settings).contramap[ByteString](bytes => OutgoingMessage(bytes, false, false, None))
 
+  def replyTo(settings: AmqpReplyToSinkSettings): Sink[OutgoingMessage, Future[Done]] =
+    Sink.fromGraph(new AmqpReplyToStage(settings))
+
   /**
-   * Scala API: Creates an [[AmqpSink]] that accepts [[OutgoingMessage]] elements.
+   * Scala API:
+   *
+   * Connects to an AMQP server upon materialization and sends incoming messages to the server.
+   * Each materialized sink will create one connection to the broker. This stage sends messages to
+   * the queue named in the replyTo options of the message instead of from settings declared at construction.
+   *
    */
-  def apply(settings: AmqpSinkSettings): Sink[OutgoingMessage, NotUsed] =
+  def apply(settings: AmqpSinkSettings): Sink[OutgoingMessage, Future[Done]] =
```

Comment:
Just document in what circumstances that `Future[Done]` is completed/failed in the scaladoc.